packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -87,6 +87,18 @@ describe('<TreeItem />', () => {
       .to.equal('endIcon');
   });
 
+  it('should allow conditional child', () => {
+    expect(() =>
+      render(
+        <TreeView defaultExpanded={['1']}>
+          <TreeItem nodeId="1" label="1" data-testid="1">
+            {false && <TreeItem nodeId="2" label="2" data-testid="2" />}
+          </TreeItem>
+        </TreeView>,
+      ),
+    ).to.not.throw();
```

Comment:
It would be awesome if you could be more specific about the expected output. For example should `nodeId="1"` be considered expanded or is it a leaf node?

Reply:
The test is specific - expected output is irrelevant.  I can remove defaultExpanded if it is confusing things.

Reply:
or remove it altogether.

Reply:
@tonyhallett Did you accidentally resolve this comment? Not sure how this was addressed.

Reply:
Have mirrored the test in the TreeView.  Apologies for the addiitonal prettier commit